mcf-browser/src/back-end/utils.js

```diff
@@ -70,8 +70,8 @@ async function getRemotePropertyLabels(dcid) {
 async function getRemotePropertyValues(dcid, label, isInverse) {
   const direction = isInverse ? 'in' : 'out';
   const targetUrl =
-      (API_ROOT + '/v1/property/' + direction + '/' + label +
-      '/values/' + dcid);
+      (API_ROOT + '/v1/property/values/' + direction + '/' + dcid +
```

Comment:
nit: can use template literals for better readability: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Changed in next commit!